data-prepper-plugins/opensearch/src/main/resources/index-template/otel-v1-apm-span-index-standard-template.json

```diff
@@ -0,0 +1,150 @@
+{
+  "version": 1,
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "dynamic_templates": [
+        {
+          "resource_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "resource.attributes.*"
+          }
+        },
+        {
+          "span_attributes_map": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "span.attributes.*"
+          }
+        }
+      ],
+      "_source": {
+        "enabled": true
+      },
+      "properties": {
+        "traceId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "spanId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "parentSpanId": {
+          "ignore_above": 256,
+          "type": "keyword"
+        },
+        "name": {
+          "ignore_above": 1024,
+          "type": "keyword"
+        },
+        "traceState": {
+          "ignore_above": 1024,
+          "type": "keyword"
+        },
+        "traceGroup": {
+          "ignore_above": 1024,
+          "type": "keyword"
+        },
+        "traceGroupFields": {
```

Comment:
It would actually be nice to add (resource) attributes to the trace group in a future extension.